examples/build.provenance

```diff
@@ -0,0 +1,205 @@
+                "svn_url": "https://github.com/slsa-framework/github-actions-demo",
+                "tags_url": "https://api.github.com/repos/slsa-framework/github-actions-demo/tags",
+                "teams_url": "https://api.github.com/repos/slsa-framework/github-actions-demo/teams",
+                "trees_url": "https://api.github.com/repos/slsa-framework/github-actions-demo/git/trees{/sha}",
+                "updated_at": "2021-06-14T17:44:26Z",
+                "url": "https://api.github.com/repos/slsa-framework/github-actions-demo",
+                "watchers": 0,
+                "watchers_count": 0
+              },
+              "sender": {
+                "avatar_url": "https://avatars.githubusercontent.com/u/1304826?v=4",
+                "events_url": "https://api.github.com/users/loosebazooka/events{/privacy}",
+                "followers_url": "https://api.github.com/users/loosebazooka/followers",
+                "following_url": "https://api.github.com/users/loosebazooka/following{/other_user}",
+                "gists_url": "https://api.github.com/users/loosebazooka/gists{/gist_id}",
+                "gravatar_id": "",
+                "html_url": "https://github.com/loosebazooka",
+                "id": 1304826,
+                "login": "loosebazooka",
+                "node_id": "MDQ6VXNlcjEzMDQ4MjY=",
+                "organizations_url": "https://api.github.com/users/loosebazooka/orgs",
+                "received_events_url": "https://api.github.com/users/loosebazooka/received_events",
+                "repos_url": "https://api.github.com/users/loosebazooka/repos",
+                "site_admin": false,
+                "starred_url": "https://api.github.com/users/loosebazooka/starred{/owner}{/repo}",
+                "subscriptions_url": "https://api.github.com/users/loosebazooka/subscriptions",
+                "type": "User",
+                "url": "https://api.github.com/users/loosebazooka"
+              },
+              "workflow": ".github/workflows/example-publish.yml"
+            },
+            "event_name": "workflow_dispatch",
+            "event_path": "/home/runner/work/_temp/_github_workflow/event.json",
+            "head_ref": "",
+            "job": "generate-provenance",
+            "ref": "refs/heads/main",
+            "repository": "slsa-framework/github-actions-demo",
+            "repository_owner": "slsa-framework",
+            "run_id": "936659519",
+            "run_number": "2",
+            "sha": "2b67dfb4e7b0bd04977d975afdafe0c76138aa1b",
+            "workflow": "Create a sample provenance",
+            "workspace": "/home/runner/work/github-actions-demo/github-actions-demo"
+          },
+          "runner": {
+            "os": "Linux",
+            "temp": "/home/runner/work/_temp",
+            "tool_cache": "/opt/hostedtoolcache"
+          }
+        }
+      },
+      "materials": [
+        {
+          "uri": "https://github.com/slsa-framework/github-actions-demo",
```

Comment:
This should use ResourceURI https://github.com/in-toto/attestation/blob/main/spec/field_types.md#field-type-definitions  which suggests using SPDX Download Location.  E.g. 'git+https'  https://spdx.github.io/spdx-spec/3-package-information/#37-package-download-location